src/blister.c

```diff
@@ -1168,8 +1165,6 @@ Obj FuncBLIST_LIST (
         if ( l * lenSub < 2 * lenList ) {
 
             /* allocate the boolean list and get pointer                   */
-            blist = NewBag( T_BLIST, SIZE_PLEN_BLIST( lenList ) );
```

Comment:
Aren't the `lenSub` and `lenList` computations a few lines above redundant, too? (not that it matters :-)

Reply:
This whole function could maybe do with a nice cleanup, but to start with I wanted to do the minimum amount to convince myself they are the same, and merge them together :)

Reply:
Of course that makes perfect sense (which is why I approved this already :-)